hal/src/main/native/include/MockData/DriverStationData.h

```diff
@@ -69,6 +70,11 @@ void HALSIM_SetDriverStationMatchTime(double matchTime);
 
 void HALSIM_NotifyDriverStationNewData(void);
 
+void HALSIM_SetJoystickAxes(int32_t joystickNum, const HAL_JoystickAxes& axes);
```

Comment:
These are C functions, so they definitely can't be references. They need to be pointers.